pkg/tsdb/influxdb/query.go

```diff
@@ -78,7 +78,24 @@ func (query *Query) renderTags() []string {
 	return res
 }
 
+func isTimeRangeNumeric(tr *plugins.DataTimeRange) bool {
+	if _, err := strconv.ParseInt(tr.From, 10, 64); err != nil {
+		return false
+	}
+	if _, err := strconv.ParseInt(tr.To, 10, 64); err != nil {
+		return false
+	}
+	return true
+}
+
 func (query *Query) renderTimeFilter(queryContext plugins.DataQuery) string {
+	// If from expressions
+	if isTimeRangeNumeric(queryContext.TimeRange) {
+		from, to, _ := epochMStoInfluxTime(queryContext.TimeRange)
```

Comment:
Shouldn't the error be handled?

Reply:
Right forgot about that. These don't return errors themselves, but I imagine I should at least log?

Reply:
@kylebrandt whatever's appropriate, I don't know the context :)